modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -921,10 +920,16 @@ You can have properties in both node data files and relationship data files.
 Data types::
 Use one of `int`, `long`, `float`, `double`, `boolean`, `byte`, `short`, `char`, `string`, `point`, `date`, `localtime`, `time`, `localdatetime`,
 `datetime`, and `duration` to designate the data type for properties.
+By default, types (except arrays) will be converted to Cypher types [https://neo4j.com/docs/cypher-manual/current/values-and-types/property-structural-constructed/#_property_types].
+This behavior can be disabled using the option `--normalize-types=false`.
+Normalizing types can require more space on disk, but avoids Cypher converting the type during queries.
 If no data type is given, this defaults to `string`.
+
 To define an array type, append `[]` to the type.
 By default, array values are separated by `;`.
 A different delimiter can be specified with `--array-delimiter`.
+Arrays are affected by the `--normalize-types` option. If you want a byte array to be stored as a Cypher long array you must explicitly declare the property as `long[]`.
```

Comment:
```suggestion
Arrays are affected by the `--normalize-types` option.
If you want a byte array to be stored as a Cypher `long` array, you must explicitly declare the property as `long[]`.
```

Reply:
I should have written: "Arrays are not affected...". I accepted @brunobuss suggestion further down that corrects this but undoes the formatting changes @renetapopova made here.